crates/client/rpc/src/lib.rs

```diff
@@ -1868,3 +1873,599 @@ fn actual_resources_to_execution_resources(resources: ResourcesMapping) -> Execu
         keccak_builtin_applications: *resources.get("keccak_builtin").unwrap_or(&0),
     }
 }
+
+// Newtype Wrapper to escape Arc orphan rules
```

Comment:
Can you just put all this in a separate module, this file is already too long :)